htdocs/api/v0.0.3-dev/candidates/visits/images/headers/Headers.php

```diff
@@ -70,22 +70,29 @@ public function handleGET()
         $SeriesName        =  $this->getHeader("acquisition:protocol");
         $SeriesDescription = $this->getHeader("acquisition:series_description");
 
-        $XSpace     = [
-                       "Length"   => $this->getHeader("xspace:length"),
-                       "StepSize" => $this->getHeader("xspace:step"),
-                      ];
-        $YSpace     = [
-                       "Length"   => $this->getHeader("yspace:length"),
-                       "StepSize" => $this->getHeader("yspace:step"),
-                      ];
-        $ZSpace     = [
-                       "Length"   => $this->getHeader("zspace:length"),
-                       "StepSize" => $this->getHeader("zspace:step"),
-                      ];
-        $TimeD      = [
-                       "Length"   => $this->getHeader("time:length"),
-                       "StepSize" => $this->getHeader("time:step"),
-                      ];
+        $XSpace = [
+                   "Length"   => $this->getHeader("xspace:length"),
+                   "StepSize" => $this->getHeader("xspace:step"),
+                  ];
+        $YSpace = [
+                   "Length"   => $this->getHeader("yspace:length"),
+                   "StepSize" => $this->getHeader("yspace:step"),
+                  ];
+        $ZSpace = [
+                   "Length"   => $this->getHeader("zspace:length"),
+                   "StepSize" => $this->getHeader("zspace:step"),
+                  ];
+        $TimeD  = [
+                   "Length"   => $this->getHeader("time:length"),
+                   "StepSize" => $this->getHeader("time:step"),
+                  ];
```

Comment:
Look like I skipped these array in my previous review,  $xSpace, $ySpace, $zSpace, $timeD.
would be great if you can change in this PR but will not block for that.

Reply:
@PapillonMcGill just renamed all variables inside this function to start with lower case. Thanks!